content/python/suspend_virtual_server.py.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Working with suspend virtual server"
+description: "A few examples on interacting with suspend virtual server."
+date: "2019-04-11"
+classes: 
+    - "SoftLayer_Product_Package"
+    - "SoftLayer_Product_Order"
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Location"
+tags:
+    - "Virtualserver"
+    - "Placeorder"
+    - "Package"
+    - "Location"  
+---
+
+###Create Suspend Virtual Server
```

Comment:
For the "Create" section, can you add an example of ordering this from the SLCLI, and one example of ordering with the ordering manager, as I think those will be a little more approachable. 

Titles something like this:
```
###Create Suspend Virtual Server
####SLCLI
####Ordering Manager
####Advanced
(what you have already  would go here)
```

Reply:
Done